.vale/styles/Localization/Modals.yml

```diff
@@ -0,0 +1,15 @@
+---
+extends: existence
+ignorecase: true
+level: warning
+link: https://redhat-documentation.github.io/vale-at-red-hat/docs/main/reference-guide/termswarnings/
+message: "Avoid using modals because they can introduce uncertainty"
```

Comment:
Suggested edit:

"Avoid using modals like 'should', 'may', or 'will' because they can introduce uncertainty."

Reply:
Thanks! Updated as suggested!